xml/importexport.xsd

```diff
@@ -14,7 +14,7 @@
 	<!-- Identifies a particular I18N locale, e.g. en -->
 	<simpleType name="locale">
 		<restriction base="string">
-			<pattern value="[a-z][a-z]_[A-Z][A-Z](@[a-z]+)?" />
+			<pattern value="([a-z]{2})((_[A-Z]{2})?)(@[a-z]{0,})?" />
```

Comment:
:+1: I'll cherry-pick this in separately, thanks!

Reply:
I extended the regex slightly to allow e.g. `ckb` (Kurdish): https://github.com/pkp/pkp-lib/commit/66f096ef37dd5b25195dfafe51257c56a6f22827